ui/src/app/pages/setting/component/asset-edit/asset-edit.component.html

```diff
@@ -187,15 +188,29 @@
                 </clr-tree-node>
             </ng-template>
           </clr-tree-node>
-  
-  
-          <button type="submit" class="btn btn-primary"  (click)="save()" [clrLoading]="editassetloading" [disabled]="DcimForm.form.invalid || editassetloading">Submit</button>
+          </div>
+          
+          <div style="margin-bottom: 15px;">
+            <label style="border-bottom-width: 1px;border-bottom-style: double;width: 100%;font-size: 16px;">Advance 
+              <span style="font-size: 12px;color: darkgray;">(Advanced settings use raw data to edit JSON strings.)</span>
```

Comment:
Use raw asset data to update the asset, please use this to update the asset as the last option.